version/1.0.md

```diff
@@ -8,67 +8,62 @@ current: true
+Each time [the Company] produces source code that may help other scientists in
+their everyday work (e.g., software, data analysis, web servers), the code
+should be made publicly available, allow for community contributions, and be
+clearly accompanied by any licensing information and/or credits.
 
-If relevant for the scientific community, scientific data collected by [the
-Company] should be stored in a public repository that explicitly states on its
-licensing and credits. [The Company] should carefully pay attention to the
-repository terms & conditions before submitting data to this repository.
-Institutional repositories (such as [Zenodo](https://zenodo.org)) must be the
-primary choice, but, third-party repositories can also be considered if the
-service maintainer (company or institution) explicitly states on the future of
-stored data when that maintainer status evolves (fusion, acquisition,
-liquidation).
+Any data collected by [the Company] that is relevant to the scientific community
+should be stored in a public repository, also explicitly accompanied by any
+licensing information and/or credits. [The Company] should pay careful attention
+to a repository's terms & conditions before submitting any data. Institutional
+repositories such as [Zenodo](https://zenodo.org) are preferred; however,
+third-party repositories may also be considered, provided that the service
+maintainer clearly addresses the treatment of stored data in the event that the
+maintainer's status were to change (e.g., fusion, acquisition, liquidation).
 
-When the time to publish research results has come, [the Company] submits its
-manuscript to Open Access journals with an Open Peer Review process.
+Once research results are ready for publication, [the Company] should submit its
+manuscript to Open Access journals using an open peer review process.
 
-## Users are not the product
+## User privacy is of utmost importance
 
-[The Company] must respect the user's right to privacy. [The Company] must not 
-sell or give away data about user's use patterns, locations, or other metadata
-without explicit, informed consent from the user.
+[The Company] must respect the user's right to privacy. Data about the user
+(e.g., use patterns, location, or any other metadata) should never be sold or
+given away without explicit informed consent from the user. Additionally,
+[the Company] must never publish any analyses containing data not publicly
+available at the time of publishing.
 
-[The Company] must respect the user's right to keep their scientific data 
-private. [The Company] will never sell or give away a user's unpublished
-scientific data. [The Company] will not publish any analysis that 
-includes data which are not publically available at the time of publishing.
+All users should be treated as contributors to [the Company]'s common goods, not
+as products. Users have a right to know where their data is being stored and how
+it is being used.
```

Comment:
Add ", and how to have their data removed" to the end before the full stop, as suggested by @kazoni here: https://github.com/TailorDev/manifesto/pull/15/commits/e694bcc0ecc5b0391d26b64a7a294e089f98abee